Cxx11/stencil-sycl-usm.cc

```diff
@@ -105,19 +105,13 @@ void run(sycl::queue & q, int iterations, size_t n, size_t tile_size, bool star,
   // Allocate space and perform the computation
   //////////////////////////////////////////////////////////////////////
 
-  auto ctx = q.get_context();
-
   double stencil_time(0);
```

Comment:
`double stencil_time{0};` or even `double stencil_time{};

Reply:
yeah, the timer stuff is a mess.  it is not broken just not consistent.  it just isn't important enough to me right now to fix.